README.md

```diff
@@ -31,6 +21,16 @@ This operator was created using the
 [operator-sdk](https://github.com/operator-framework/operator-sdk/). It's not
```

Comment:
is this really a prerequisite if it is not strictly required.
maybe add an optional section?